omod/src/main/java/org/openmrs/module/webservices/rest/web/controller/SwaggerSpecificationController.java

```diff
@@ -27,45 +27,12 @@
 	public @ResponseBody
 	String getSwaggerSpecification(HttpServletRequest request) throws Exception {
 		
-		String swaggerSpecificationJSON = "";
-		StringBuilder baseUrl = new StringBuilder();
-		String scheme = request.getScheme();
-		int port = request.getServerPort();
-		
-		baseUrl.append(scheme); // http, https
-		baseUrl.append("://");
-		baseUrl.append(request.getServerName());
```

Comment:
Does this ticket involve removing all these lines of code (from line 30 - 68)?
Are you sure we are not having any regressions here?